### ISSUE TITLE
Installer for Termite

### ISSUE THREAD

**Issue by cawoodm** [webinstall/webi-installers#295]
Can we suggest a new installer by submitting a .sh file like this?

https://raw.githubusercontent.com/Corwind/termite-install/master/termite-install.sh

**Comment by coolaj86** [comment]
No. See https://github.com/webinstall/webi-installers/issues/294#issuecomment-907654551

If you can find official static builds, the the installer should follow one of the example templates, but perhaps have a custom releases.js of it’s not on GitHub.

Here’s a link to a template issue for adding an installed that has the steps listed out: https://github.com/webinstall/webi-installers/issues/187

**Comment by coolaj86** [comment]
Actually, I would potentially accept hosting static builds as well, if the maintainer refuses.

For example, if you got a process for static builds for fish (I don’t understand why the maintainer is opposed to doing it the “standard” way), I’d also be happy to add those.

I had to do this for xz, and I had to use someone else’s for ffmpeg.
